client/index.js

```diff
@@ -1,41 +1,53 @@
-import { updateBrowsersStats, updateGlobalCoverageBar, updateToolsVersions, hideStatsPlaceholder } from './view/showStats.js';
 
-const API_HOST = 'http://localhost:5000/api';
+import { updateBrowsersStats, updateRegionCounter, updateRegionCoverageBar, updateToolsVersions, hideStatsPlaceholder } from './view/showStats.js';
+import regionsList from './data/regions.js'
+import initRegionSelect from './view/region-select'
+
+const API_HOST = 'http://localhost:5000/api/';
 
 const form = document.getElementById('query_form');
 const textarea = document.getElementById('query_text_area');
 const errorMessage = document.getElementById('error_message');
+const select = document.getElementById('region-select');
+
+function initForm() {
+  let sendForm = () => form.dispatchEvent(new Event('submit'));
+
+  initRegionSelect(select, regionsList, sendForm)
 
-  function initForm() {
   form.addEventListener('submit', async (e) => {
     if(!form.checkValidity()) {
       return;
     }
     const formData = new FormData(form);
     const query = formData.get('query')
+    const region = formData.get('region')
     e.preventDefault();
     form.classList.add('Form--justSend');
     textarea.addEventListener('input', () => {
       form.classList.remove('Form--justSend');
     }, {
       once: true,
     })
-    updateStatsView(query);
+    updateStatsView(query, region);
   })
+
   textarea.addEventListener('keypress', (e) => {
     if (e.keyCode === 13 && !e.shiftKey) {
-      e.preventDefault();
-      form.dispatchEvent(new Event('submit'));
+      e.preventDefault()
+      sendForm(e)
     }
   })
 }
 
 initForm();
 
-async function updateStatsView(query) {
+async function updateStatsView(query, region) {
   let response;
   try {
-    response = await fetch(`${API_HOST}/browsers?q=${encodeURIComponent(query)}`)
+    let urlParams = new URLSearchParams({ q: query, region })
+    let url = new URL(`browsers?${urlParams}`, `${API_HOST}`)
+    response = await fetch(url)
   } catch (error) {}
```

Comment:
Don’t like that we put error processing at the end.

We should start from edge cases, since it is very easy to forget about them.

Let’s add loader and network errors in this PR since we started to work on real front-end.